app/controllers/verify/authentications_controller.rb

```diff
@@ -23,6 +23,7 @@ def create
         current_claim.update!(@response.claim_parameters)
         redirect_to claim_url("verified")
       else
+        current_claim.update!(verify_response: @response.parameters)
```

Comment:
This means we would only store unverified responses, which is not actually that interesting; they should all look something like `{ "scenario": "AUTHENTICATION_FAILED" }`. The intent of this story is to store the response for "verified" claims so we can deal with situations where, for example, we find the address we're extracted doesn't match the address on record.

Reply:
oh, just noticed that this is in addition to setting the response in `claim_parameters`. ignore!